be/src/util/brpc_stub_cache.h

```diff
@@ -34,25 +34,32 @@ class BrpcStubCache {
     BrpcStubCache();
     virtual ~BrpcStubCache();
 
-    virtual PBackendService_Stub* get_stub(const butil::EndPoint& endpoint) {
-        std::lock_guard<SpinLock> l(_lock);
+    virtual std::shared_ptr<PBackendService_Stub> get_stub(const butil::EndPoint& endpoint) {
+        std::lock_guard<std::mutex> l(_mutex);
```

Comment:
Why not using SpinLock?

Reply:
I think we should try to use the class in stl